rpcserver.go

```diff
@@ -379,6 +378,63 @@ func (s *server) Send(ctx context.Context, req *pbrpc.SendRequest) (*pbrpc.SendR
 	}, nil
 }
 
+func (s *server) ListPeer(
+	ctx context.Context,
+	req *pbrpc.RPCListPeerRequest) (*pbrpc.RPCListPeerResponse, error) {
+	peerIDs := s.node.Network().Peers()
+	return &pbrpc.RPCListPeerResponse{
+		Response: makeResponse(true, ""),
+		Peers:    peerIDsToPBPeers(peerIDs),
+	}, nil
+}
+
+func (s *server) ListTopicPeer(
+	ctx context.Context,
+	req *pbrpc.RPCListTopicPeerRequest) (*pbrpc.RPCListTopicPeerResponse, error) {
+	var topics []string
+	if len(req.Topics) == 0 {
+		topics = s.node.pubsubService.GetTopics()
+	} else {
+		topics = req.Topics
+	}
+	topicPeers := make(map[string]*pbmsg.Peers)
+	for _, topic := range topics {
+		peerIDs := s.node.pubsubService.ListPeers(topic)
+		topicPeers[topic] = peerIDsToPBPeers(peerIDs)
+	}
+	return &pbrpc.RPCListTopicPeerResponse{
+		Response:   makeResponse(true, ""),
+		TopicPeers: topicPeers,
+	}, nil
+}
+
+func (s *server) RemovePeer(
+	ctx context.Context,
+	req *pbrpc.RPCRemovePeerRequest) (*pbrpc.RPCPlainResponse, error) {
+	// Add span for StopServer
+	spanctx, err := logger.StartFromParentState(ctx, "RPCServer.RemovePeer", s.serializedSpanCtx)
+	if err != nil {
+		logger.Debugf("Failed to deserialze the trace context. Tracer won't be able to put rpc call traces together. err: %v", err)
+		spanctx = logger.Start(ctx, "RPCServer.RemovePeer")
+	}
+	defer logger.Finish(spanctx)
+
+	peerID, err := stringToPeerID(req.PeerID)
+	if err != nil {
+		errorMsg := fmt.Sprintf("failed to parse the peerID: %v", req.PeerID)
+		return makePlainResponse(false, errorMsg), nil
```

Comment:
IMO it makes more sense to `return nil, errorMsg` than `return makePlainResponse(false, errorMsg), nil` when error indeed occurs.